src/data-mgt/python/jobs/airqo-frequency-measurements/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-slim-buster
+MAINTAINER AirQo "airqo"
+
+ADD requirements.txt /
+WORKDIR /
+
+RUN pip install -r requirements.txt
+ADD calculate_measurements_by_frequency.py /
+
+ENV DEVICE_REGISTRY_URL=http
+ENV FREQUENCY=hourly
+ENV START_TIME=hourly
```

Comment:
Isn't this suppose to be a datetime value

Reply:
Yes @ja-odur, that is expected behaviour, those 504 errors have been around for quite a long time, if you can remember. The dockerfile has nothing to do with the testing, and it's not used during testing. It's only used during deployment during which those variables will be set